src/pages/404.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Button from 'react-bootstrap/lib/Button';
+import Link from 'gatsby-link';
+
+function fourOhFour() {
```

Comment:
This was my original mistake. React component declarations need to start with an uppercase char, else it will be parsed as HTML and not work as expected

So this should be `FourOhFour`